docs/index.md

```diff
@@ -367,6 +367,27 @@ Adding errors to action object is important, because each time a handled excepti
 Validation exception will have the same backtrace as original error.
 Prefer this way over custom exception handling in private methods.
 
+### Dependency Injection
+
+Granite uses action initializer for attributes.
```

Comment:
I don't really understand what you mean here? What is "action initializer"?

Reply:
@andriusch `MyBa#initialize`.

Reply:
Hm maybe reword this to something like "Granite assigns attributes in `initialize` method based on parameters that are passed to the initializer".